source/topics/paas/how-to-setup-application-monitoring.md

```diff
@@ -0,0 +1,9 @@
+Aptible recommends New Relic for application monitoring, and now supports it at the [host-level](https://github.com/aptible/support/issues/115) as well.
+
+(_Note: When running the New Relic Server Monitoring product in combination with the Application Monitoring product, customers are only charged per host instead of per app process._)
+
+To have it added to your stack, simply contact support with the following: 
```

Comment:
These steps are unclear to me (why am I setting the New Relic API license key? why do I have to choose an app? isn't this going to monitor all my apps?) Maybe something like the following would be better:

> To request New Relic Server Monitoring for your PHI-ready environment, simply contact support and let us know your New Relic license key. You can securely communicate your license key to us by either:

> * Sending a PGP-encrypted email to support@aptible.com. Our public key is registered and also available [on our Responsible Disclosure page](https://www.aptible.com/legal/responsible_disclosure.html).
> * Setting the key as an ENV variable on one of your apps: `aptible config:set NEW_RELIC_LICENSE_KEY=...`